client/landing/stepper/declarative-flow/flows/site-setup-flow/site-setup-flow.ts

```diff
@@ -272,6 +275,9 @@ const siteSetupFlow: FlowV1 = {
 		} );
 
 		useRedirectDesignSetupOldSlug( currentStep, navigate );
+		const { get } = useFlowState();
+		const entryPoint = get( 'flow' )?.entryPoint;
+		const siteAdminUrl = useSelector( ( state ) => getSiteAdminUrl( state, siteId ) );
```

Comment:
I see that `adminUrl` is defined above. Is there a reason we need a second selector here?

Reply:
Thanks, removed here [78bd9f4](https://github.com/Automattic/wp-calypso/pull/102224/commits/78bd9f4191b5c19b5aea1d9490a06eed88381b67)